deps/rabbitmq_stomp/src/rabbit_stomp_processor.erl

```diff
@@ -887,38 +1198,32 @@ ensure_reply_to(Frame = #stomp_frame{headers = Headers}, State) ->
             end
     end.
 
-ensure_reply_queue(TempQueueId, State = #proc_state{channel       = Channel,
-                                               reply_queues  = RQS,
-                                               subscriptions = Subs}) ->
+ensure_reply_queue(TempQueueId, State = #proc_state{reply_queues  = RQS,
+                                                    subscriptions = Subs}) ->
     case maps:find(TempQueueId, RQS) of
         {ok, RQ} ->
             {binary_to_list(RQ), State};
         error ->
-            #'queue.declare_ok'{queue = Queue} =
-                amqp_channel:call(Channel,
-                                  #'queue.declare'{auto_delete = true,
-                                                   exclusive   = true}),
+            {ok, Queue} = create_queue(State),
+            #resource{name = QNameBin} = QName = amqqueue:get_name(Queue),
 
             ConsumerTag = rabbit_stomp_util:consumer_tag_reply_to(TempQueueId),
-            #'basic.consume_ok'{} =
-                amqp_channel:subscribe(Channel,
-                                       #'basic.consume'{
-                                         queue        = Queue,
-                                         consumer_tag = ConsumerTag,
-                                         no_ack       = true,
-                                         nowait       = false},
-                                       self()),
-
-            Destination = binary_to_list(Queue),
+            Spec = #{no_ack => true,
+                     prefetch_count => application:get_env(rabbit, default_consumer_prefetch),
```

Comment:
Wrong prefetch count.
It leads to crashes later on.

e.g.
```
SEND
destination:/queue/reply-test
reply-to:/temp-queue/foo

Hello World!^@
```
then click on the reply-to-queue in the Management UI. You get a 500 error and RabbitMQ logs:
```
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>   crasher:
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>     initial call: cowboy_stream_h:request_process/3
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>     pid: <0.1278.0>
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>     registered_name: []
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>     exception error: no function clause matching
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>                      thoas_encode:value({ok,{false,0}},
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>                                         #Fun<thoas_encode.0.30747453>) (src/thoas_encode.erl, line 1720)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in function  thoas_encode:map_naive_loop/2 (src/thoas_encode.erl, line 1712)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from thoas_encode:map_naive_loop/2 (src/thoas_encode.erl, line 1713)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from thoas_encode:map_naive/2 (src/thoas_encode.erl, line 1704)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from thoas_encode:list/2 (src/thoas_encode.erl, line 1692)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from thoas_encode:map_naive/2 (src/thoas_encode.erl, line 1703)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from thoas:encode/2 (src/thoas.erl, line 92)
2023-09-06 14:10:00.942131+00:00 [error] <0.1278.0>       in call from rabbit_mgmt_util:reply0/3 (rabbit_mgmt_util.erl, line 252)
```